code/modules/mob/living/devour.dm

```diff
@@ -44,6 +46,16 @@
 		src << "\red You can't eat yourself!"
 		return 0
 
+	if (devouring == victim)
```

Comment:
People are stupid and you can misclick. Stopping should be an affirmative and separate action. Until you do that, or finish, it should deny starting on another.